### ISSUE TITLE
Roll-back instructions

### ISSUE THREAD

**Issue by scottmetoyer** [hudmol/container_management#108]
I inherited an ArchivesSpace installation that had this plugin installed. The client asked to have the plugin removed.

We backed everything up and went ahead and removed it by reversing the installation instructions, but we are now receiving errors when attempting to create new resources:

```
KeyError in Resources#new

Showing C:/inetpub/wwwroot/archivespace/data/tmp/jetty-0.0.0.0-8080-frontend.war-_-any-/webapp/WEB-INF/app/views/resources/new.html.erb where line #5 raised:

key not found: restriction_type
```

Can you provide instructions on properly removing the plugin?

We are on ArchivesSpace v. 1.4.2.

**Comment by jambun** [comment]
Hi @scottmetoyer - Rollback wasn't a requirement for this project, and the plugin makes fairly significant changes to the data model, so it may be non-trivial. It does aim to maintain compatibility with the old container model however, so it should be possible. Note also that this functionality was merged into the core application in the v1.5 release.  Cheers -- James

**Comment by scottmetoyer** [comment]
Hi James,

Thanks for the quick response!

I went ahead and just reinstalled the plugin which did correct the errors. We are planning to upgrade to 1.5 in the next few weeks.

Really appreciate the help - Scott
